l10n_it_financial_statement_eu/static/src/js/client_action.esm.js

```diff
@@ -0,0 +1,48 @@
+/** @odoo-module **/
+import {ReportAction} from "@web/webclient/actions/reports/report_action";
+import {patch} from "web.utils";
+// Import { patch } from '@web/core/utils/patch';
+import {useEnrichWithActionLinks} from "./report.esm";
+
+const MODULE_NAME = "l10n_it_financial_statement_eu";
+
+patch(ReportAction.prototype, "l10n_it_financial_statement_eu.ReportAction", {
```

Comment:
In questo modo vengono modificati i metodi dell'azione generica per produrre dei report, quindi in pratica ogni report avrà questi nuovi metodi `export_xlsx`, è davvero il comportamento desiderato?
Si potrebbe invece dichiarare una nuova classe che estende `ReportAction` e usare quella nei report, cosa ne pensi?

Altrimenti, potresti almeno dare dei nomi più specifici a questi metodi?
Così si evitano conflitti tipo quello di `_get_xlsx_name` segnalato in https://github.com/OCA/l10n-italy/pull/3599#pullrequestreview-1922867564.

Reply:
Ho messo nomi specifici perchè creando una nuova classe che estende la ReportAction poi non saprei come dire al report di usare la nuova classe